docker/devbox/fluent.conf

```diff
@@ -0,0 +1,84 @@
+<source>
+  	@type tcp
+  	tag mediawiki
+  	port 9999
+  	<parse>
+		@type json
+		time_format %Y-%m-%dT%H:%M:%S.%N%z
+		time_key @timestamp
+	</parse>
+</source>
+
```

Comment:
with syslog-ng, we are throttling the messages: https://github.com/Wikia/chef-repo/blob/37e07590b35b632eb53005ffbb9bc8bb2bb5fb70/cookbooks/syslog-ng-18/attributes/default.rb#L12. What do you think about using something like https://github.com/rubrikinc/fluent-plugin-throttle to limit the messages on dev (in case of some infinite loops or similar)?

Reply:
Done @themech